InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -117,6 +121,88 @@ public void robotInit() {
         lastTime = System.currentTimeMillis();
     }
 
+    public void wireUpButtons() {
+
+        Buttons b = new Buttons();
+
+        //////////////////////////////////////////////
+        // Drive Subsystem
+        if (config.enableDriveSubsystem) {
+            b.driveAutoAim.whenHeld(new InstantCommand(() -> driveSubsystem.setAutoAligning(true), driveSubsystem)).whenReleased(new InstantCommand(() -> driveSubsystem.setAutoAligning(false), driveSubsystem));

```

Comment:
Rather than setting auto aligning, we have an auto align command, we can just say `b.driveAutoAim.whenHeld(new AutoAlignCommand())`. This way when you hold down the autoAlign button, it will replace the default command with the autoAlign command which executes the auto align logic. When the button is released it should cancel the command and return to the default.